### ISSUE TITLE
🐛 [BUG]

### ISSUE THREAD

**Issue by rishu2020** [e3nn/e3nn#368]
**Describe the bug**

For the MNIST dataset that you've provided, the output from S2conv_original(..) has negative values. Aren't these supposed to be probabilities? Which means, they should be between 0 and 1?

Just after that, the prediction turns out to be: [c, c, c, c, c, ..., c], where c = an integer in the range(0, num_classes)

Which is obviously not correct? I may have limited understanding ... 

On the MNIST dataset, the test accuracy comes out to be ~10% ... which is again very low.

It's possible that I'm missing something.

**To Reproduce**

**Expected behavior**

**Environment (please complete the following information):**
 - OS: [e.g. iOS, Ubuntu, Windows]
 - python version (`python --version`)
 - python environment (commands are given for python interpreter):

**Screenshots**

**Additional context**

**Comment by mariogeiger** [comment]
The output of the network is given to [`TORCH.NN.FUNCTIONAL.CROSS_ENTROPY`](https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html#torch.nn.functional.cross_entropy) which takes as argument so called `logits` that are not yet probabilities.
The `cross_entropy` function will apply softmax to convert those logits to probabilities (it will in practice not do it because the loss is the negative log likelihood and then the conversion to probabilities and the log operation are fused into a single operation).
